webview.go

```diff
@@ -50,6 +50,16 @@ const (
 	HintMax = C.WEBVIEW_HINT_MAX
 )
 
+type Status int
+
+const (
+	// Binding call succeeded
+	StatusSuccess = 0
+
+	// Binding call failed
+	StatusError = 1
```

Comment:
`-1` was used previously but other bindings seemed to be using `1` to state an error.